sage-ios/SAGE/SAGE/AnnouncementsDetailView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  AnnouncementsDetailView.swift
+//  SAGE
+//
+//  Created by Erica Yin on 11/7/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class AnnouncementsDetailView: UIView {
+    var tableView = UITableView()
```

Comment:
Make this a scrollview not a tableView. Then, in init, say scrollView.alwaysBounceVertical = true